src/approaches/reinforcement_learning_approach.py

```diff
@@ -0,0 +1,91 @@
+"""A bilevel planning approach that learns NSRTs from an offline dataset, and
+continues learning options through reinforcement learning."""
+
+from typing import Callable, List, Sequence, Set
+
+from gym.spaces import Box
+
+from predicators.src import utils
+from predicators.src.approaches.base_approach import ApproachFailure, \
+    ApproachTimeout
+from predicators.src.approaches.nsrt_learning_approach import \
+    NSRTLearningApproach
+from predicators.src.settings import CFG
+from predicators.src.structs import NSRT, Dataset, GroundAtom, \
+    InteractionRequest, InteractionResult, LowLevelTrajectory, \
+    ParameterizedOption, Predicate, State, Task, Type
+
+
+class ReinforcementLearningApproach(NSRTLearningApproach):
```

Comment:
let's rename this to NSRTReinforcementLearningApproach since it's NSRT-specific (and that better reflects the name we've given it, which is `nsrt_rl`)

Reply:
(rename the file too)